packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -21,15 +21,12 @@ const defaultTopMatchesLimit = 10;
 int topMatchesLimit = defaultTopMatchesLimit;
 
 mixin SearchControllerMixin<T extends DataSearchStateMixin> {
-  // Initial values for searching (richness primarily auto-complete).
-  TextEditingValue searchTextFieldValue = const TextEditingValue();
-
   final _searchNotifier = ValueNotifier<String>('');
 
   /// Notify that the search has changed.
   ValueListenable get searchNotifier => _searchNotifier;
 
-  bool isField = false;
+  bool get isField => search.endsWith('.');
```

Comment:
this seems specific to the debugger and not something that should be in a general Search mixin. Can we move this to the debugger?